xml/docu_styleguide.language.xml

```diff
@@ -259,11 +259,12 @@
    </listitem>
    <listitem>
     <para>
-     Use a trailing slash to distinguish between directories (for example,
-     <filename>/etc/YaST2/</filename>) and files (for example,
-     <filename>/etc/YaST2/control.xml</filename>).
-     If the distinction is important in the current context, in
-     addition, comment on it in the running text.
+     For directories, add a trailing slash to distinguish between a directory
+     (for example, <filename>/etc/YaST2/</filename>) and a file (for example,
+     <filename>/etc/YaST2/control.xml</filename>). For less experienced Linux
```

Comment:
The sentence beginning ("For directories,...") and the end of the sentence ("...and a file") don't really go together.
Suggestion:

```suggestion
     When referencing a directory name, add a trailing slash.
     This helps distinguish between directory names (for example,
     <filename>/etc/YaST2/</filename>) and file names (for example,
     <filename>/etc/YaST2/control.xml</filename>). For less experienced Linux
```

(Otherwise, this let's go to market.)